### PR TITLE
Add setup.py for easier install

### PR DESCRIPTION
**Pull request by fabianp**
Hi,

I thought it would be easier to install if it had a setup.py (it wasn't clear for me where pelican_bibtex.py should be dropped). On top of that it would be great if you could upload it to the PYPI :-)

### CODE REVIEW

**Comment by fabianp** [comment]
ping @vene . Just in case you don't get notified.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Programming Language :: Python :: 3
+Topic :: Software Development
+Operating System :: POSIX
+Operating System :: Unix
+
+"""
+
+setup(
+    name='pelican_bibtex',
+    description='A module for estimating Hemodynamical Response Function from functional MRI data',
```
Comment (vene) [inline setup.py]: Wow, sounds cool but a bit out of my area of expertise ;)

Comment (fabianp) [inline setup.py]: oops, that's what happens when you copy/paste without looking. Fixed

**Comment by vene** [comment]
Was trickier than I expected, but merged. Thanks a lot!

**Comment by tantignac** [comment]
Hello,

The `_`character in the name of the package seems to cause some problems to pip:

```
> pip install pelican_bibtex
Downloading/unpacking pelican-bibtex
  Could not find any downloads that satisfy the requirement pelican-bibtex
No distributions at all found for pelican-bibtex
```

The problem seems to have already been encountered (http://mail.python.org/pipermail//bangpypers/2011-December/007382.html and http://code.google.com/p/surfrdf/issues/detail?id=56).

**Pull request merged**